packages/contracts/contracts/core/acl/ACL.sol

```diff
@@ -63,14 +63,32 @@ contract ACL is Initializable {
     mapping(bytes32 => bool) internal freezePermissions;
 
     // Events
+
+    /// @notice Emitted when a new permission `role` is granted to the address `actor` by the address `who` in the contract `where`
+    /// @param role The hash of the role identifier
+    /// @param actor The address receiving the new role
+    /// @param who The address (EOA or contract) owning the permission
+    /// @param where The address of the contract    
+    /// @param oracle The ACLOracle to be used or it is just the ALLOW_FLAG
```

Comment:
`or it is just the ALLOW_FLAG`
clarify by reformulating the sentence. Maybe a @dev note should be added.

Reply:
How would you rephrase the sentence so that it could be better understood? I'm not sure...

Reply:
We should add a `@dev` or `@notice` (we don't use them uniformly in `ACL.sol`) comment to the `_grant` function and or `_grantWithOracle` function explaining that the`_oracle` parameter  is used to either set `authPermissions[permission]` to 
the value of`ALLOW_FLAG` to give permission or an `IACLOracle` address to redirect the permission request to an `IACLOracle` implementation.

Reply:
Should I make the addition myself @jpaulet ?

Reply:
@Michael-A-Heuer yes, go ahead. I think it's best to speed up & not use more of your time with this back-n-forth. Thanks